bitnami/mongodb/templates/replicaset/scripts-configmap.yaml

```diff
@@ -254,28 +254,34 @@ data:
       # read rs.conf again and store it. settings format is '"<key>" : <value>,'
       currentRsConf=$(mongosh ${usernameAndPassword} --eval 'rs.conf()')
 
-      desiredEqualsactual=unknown
+      desiredEqualsActual=unknown
       settingsToConfigure=""
       for key in ${!desiredRsConf[@]}; do
         value=${desiredRsConf[$key]}
-        if ! $(echo "\"${currentRsConf}"\" | grep -q -e "${key}: ${value},"); then
```

Comment:
This was the problematic line: per-member settings would not appear in `currentRsConf` in a way that grep would find. The script thus looped endlessly, always trying to set the same setting.